pennylane/pytrees.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2018-2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+An internal module for working with pytrees.
+"""
+
+from typing import Callable, Tuple, Any
+
+has_jax = True
+try:
+    import jax.tree_util as jax_tree_util
```

Comment:
Is there a way we can avoid importing JAX with PennyLane?

Reply:
No. 

And i saw no import time difference.

Reply:
This is a shame, it locks us in to having to import JAX at the same time as PennyLane.